lra/api/1.0/openapi.json

```diff
@@ -0,0 +1,922 @@
+{
+  "openapi" : "3.0.3",
```

Comment:
personally I've never seen OpenAPI doc in JSON format, it was always YAML. Why did you choose JSON?

Reply:
Also I suppose, this is generated from the code, there is no need to verify it line by line right?

Reply:
yes, this is generated from the code. The Yaml and Json format are generated both from the source by mvn plugin
https://github.com/jbosstm/narayana/pull/1813/files